assignments/assignment1/KNN.ipynb

```diff
@@ -30,6 +30,8 @@
    "outputs": [],
    "source": [
     "import numpy as np\n",
+    "np.random.seed(42) # makes the random numbers predictable\n",
```

Comment:
Let's use it closer to actual usage in shuffling, otherwise any random invocation in the middle can screw it up